Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM itkdev/php8.3-fpm:latest
+
+USER root
+
+# Add rsync
+RUN apt-get update && apt-get --yes install rsync && apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
```

Comment:
Consider moving the cleanup stuff (`apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*`) to a `RUN` statement after installing everything.